docs/using-kontena/nodes.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Nodes
+---
+
+# Nodes
+
+Each node is a machine running the `kontena-agent`. The nodes connect to the Kontena Master, which then schedules and deploys services onto those nodes.
+
+## Connecting nodes
+
+Nodes do not need to be explicitly created. The Kontena Master will automatically create new grid nodes for any `kontena-agent` connecting to the Kontena Master with the correct [grid token](grids#Grid Token).  
+
+## Online nodes
+
+Nodes will be considered as online so long as they have an active Websocket connection to the Kontena Master.
+Grid service instances can be deployed to any online node, unless restricted using [service affinity filters](deploy#Affinity) or the [grid default affinity](grids#Default Affinity).
+
+The `kontena node rm` command can not be used to remove online nodes. The node must first be terminated (using the `kontena <provider> node terminate` command), and can then be removed once offline.
+
+## Offline nodes
+
+If the agent's Websocket connection to the master is disconnected or times out, the server will mark the nodes as as offline.
+
+Offline nodes will not have any new service instances scheduled to them. Any services with instances deployed to any offline nodes will be re-scheduled by the server, moving the instances to the remaining online nodes.
+
```

Comment:
Right... and the `NodePlugger` doesn't immediatel reschedule services either, anymore.